.github/workflows/reviews.yml

```diff
@@ -0,0 +1,26 @@
+name: "Reviews"
+
+on:
+  pull_request_review:
+    types: [dismissed]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  cherry-pick:
+    name: Resolve dismissed CI reviews
+    if: github.event.review.user.login == 'github-actions[bot]'
+    runs-on: ubuntu-24.04-arm
+    steps:
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
+        with:
+          script: |
+            await github.graphql(`mutation($node_id:ID!) {
+              minimizeComment(input: {
+                classifier: RESOLVED,
+                subjectId: $node_id
+              })
+              { clientMutationId }
+            }`, { node_id: context.payload.review.node_id })
```

Comment:
Does this actually need JS? Would the workflow be any more efficient by using the `gh` CLI tool to send the request instead?

```yml
- env:
    node_id: ${{ github.event.review.node_id }}
  run: |
    gh api graphql -F node_id="$node_id" -f query='
      mutation($node_id:ID!) {
        minimizeComment(input: {
          classifier: RESOLVED,
          subjectId: $node_id
        })
        { clientMutationId }
      }
    '
```

_(I don't think I converted the context path correctly, but as a rough example...)_

If it's just a style choice and there's no downside to either approach, feel free to ignore this comment!

Or maybe you prefer this style to accommodate future expansion of the workflow?

Reply:
Thanks for explaining!